src/main/migrations/changesets/call_result_caching.xml

```diff
@@ -2,11 +2,12 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
-
+    <property name="boolean.type" value="BOOLEAN" dbms="oracle,h2,hsqldb"/>
```

Comment:
I don't believe there's precedent for dbms other than hsqldb and mysql in our changesets, this should maybe just be !mysql